packages/connect/src/protocol/is-protobuf-message.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Message } from "@bufbuild/protobuf";
+
+/**
+ * Temporary workaround for dual package hazard caused by @bufbuild/protobuf
+ *
+ * @private Internal code, does not follow semantic versioning.
+ */
+export function isProtobufMessage<T extends Message<T>>(
```

Comment:
This is a temporary workaround for the dual package hazard caused by protobuf-es. Since the ESM wrapper delegates to CJS, we're internally using `Message` from protobuf-es in CJS, but the tests are providing `Message` from protobuf-es in ESM.

I don't want to land this - we have to fix and release protobuf-es first, than revert https://github.com/connectrpc/connect-es/pull/842/commits/b57115ce70005b256530815db2917fb6e4604984 here.

Reply:
Updated to [@bufbuild/protobuf v1.3.3](https://github.com/bufbuild/protobuf-es/releases/tag/v1.3.3) in https://github.com/connectrpc/connect-es/pull/842/commits/567b5c99933daf62175ac4cbe1e74db874842025.

Reverted this temporary workaround in https://github.com/connectrpc/connect-es/pull/842/commits/598f1e40ae24b770366677e067680c090e80cd60.

Reply:
Repeated the tests from https://github.com/connectrpc/examples-es/pull/1002 successfully.